packages/scene-composer/src/components/panels/scene-components/AnchorComponentEditor.tsx

```diff
@@ -60,17 +55,22 @@ export const AnchorComponentEditor: React.FC<IAnchorComponentEditorProps> = ({
   const { listSceneRuleMapIds } = useSceneDocument(sceneComposerId);
   const intl = useIntl();
 
+  const dynamicSceneEnabled = useDynamicScene();
+
   const ruleMapIds = listSceneRuleMapIds();
   const selectedRuleMapId =
     anchorComponent.ruleBasedMapId && ruleMapIds.includes(anchorComponent.ruleBasedMapId)
       ? anchorComponent.ruleBasedMapId
       : null;
 
   const onUpdateCallbackDebounced = useCallback(
-    debounce((componentPartial: any, replace?: boolean) => {
-      const componentPartialWithRef: ISceneComponentInternal = { ref: component.ref, ...componentPartial };
-      updateComponentInternal(node.ref, componentPartialWithRef, replace);
-    }, 100),
+    debounce(
+      (componentPartial: any, replace?: boolean) => {
+        const componentPartialWithRef: ISceneComponentInternal = { ref: component.ref, ...componentPartial };
+        updateComponentInternal(node.ref, componentPartialWithRef, replace);
```

Comment:
it's hard to debounce in updateComponentInternal without additional data storage to store all requests debounced. otherwise only the last updateComponentInternal will actually be executed. So use it here will still have some problem but mostly fixed. so we need the proper solution to centralize the updates instead